taskweaver/llm/google_genai.py

```diff
@@ -0,0 +1,159 @@
+from typing import Any, Generator, List, Optional
+
+from injector import inject
+
+from taskweaver.llm.base import CompletionService, EmbeddingService, LLMServiceConfig
+from taskweaver.llm.util import ChatMessageType, format_chat_message
+
+try:
+    import google.generativeai as genai
+except Exception:
+    raise Exception(
+        "Package google-generativeai is required for using Google Gemini API. "
+        "Please install it manually by running: `pip install google-generativeai`",
+    )
```

Comment:
Maybe we should move this into `__init__` of the service class? Otherwise, such warning would bring regardless whether we have this API type configured so long as we import this file. (or we could move the import in `__init__.py`)

Reply:
fixed